packages/jaeger-ui/src/model/ddg/transformTracesToPaths.test.js

```diff
@@ -164,11 +164,10 @@ describe('transform traces to ddg paths', () => {
     const kindlessTrace = makeTrace([rootSpan, focalSpan, kindlessSpan], 'kindlessTraceID');
 
     const { dependencies: result } = transformTracesToPaths(makeTraces(clientTrace, kindlessTrace), focalSvc);
-    expect(new Set(result)).toEqual(
-      new Set([
-        makeExpectedPath([rootSpan, focalSpan], clientTrace),
-        makeExpectedPath([rootSpan, focalSpan], kindlessTrace),
-      ])
-    );
+
+    const path = makeExpectedPath([rootSpan, focalSpan], clientTrace);
+    path.attributes.push({ key: 'exemplar_trace_id', value: kindlessTrace.data.traceID });
+
+    expect(new Set(result)).toEqual(new Set([path]));
```

Comment:
perhaps this should be in a separate test (with appropriate title about deduping), not in `span.kind === 'server'` test

Reply:
Yes, well I just fixed this test to pass, but I think this test wil be removed or replaces on this PR: https://github.com/jaegertracing/jaeger-ui/pull/557 as discussed here: https://github.com/jaegertracing/jaeger-ui/issues/554 WDYT?

Reply:
even more important to have a standalone test for deduping.